src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentEventResource.java

```diff
@@ -89,6 +90,23 @@ private boolean isTextAssessmentAnalyticsEnabled() {
         return ResponseEntity.badRequest().build();
     }
 
+    /**
+     * GET events/courses/{courseId}/exercises/{exerciseId} : get the number of the tutors involved in the list of events
+     * @param courseId the id of the course to query events for
+     * @param exerciseId the id of the exercise to query events for
+     * @return an integer representing the number of tutors involved for the respective course and exercise
+     */
+    @GetMapping("events/courses/{courseId}/exercises/{exerciseId}")
```

Comment:
This is not a good REST URL

Reply:
Applied refactoring on the URL
`events/courses/{courseId}/exercises/{exerciseId}`
changed to 
`courses/{courseId}/text-exercises/{exerciseId}/tutors-involved`